test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -4810,6 +4812,97 @@ public void RestoreNetCore_NETCore_ProjectToProject_IgnoreMissingProjectReferenc
             }
         }
 
+        [Fact]
+        public void RestoreNetCore_AssetTargetFallbackWithProjectReference_VerifyFallbackToNet46Assets()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+
+                var netcoreapp2 = NuGetFramework.Parse("netcoreapp2.0");
+                var ne461 = NuGetFramework.Parse("net461");
+
+                var projectA = SimpleTestProjectContext.CreateNETCore(
+                    "a",
+                    pathContext.SolutionRoot,
+                    netcoreapp2);
+                projectA.Properties.Add("AssetTargetFallback", "net461");
+                projectA.Properties.Add("RuntimeIdentifiers", "win10");
+
+                var projectB = SimpleTestProjectContext.CreateNETCore(
+                    "b",
+                    pathContext.SolutionRoot,
+                    ne461);
+                projectA.AddProjectToAllFrameworks(projectB);
+
+                solution.Projects.Add(projectA);
+                solution.Projects.Add(projectB);
+                solution.Create(pathContext.SolutionRoot);
+
+                // Act
+                var r = Util.RestoreSolution(pathContext);
+
+                // Assert
+                r.Success.Should().BeTrue();
+                foreach (var graph in projectA.AssetsFile.Targets)
+                {
+                    var lib = graph.GetTargetLibrary("b");
+
+                    lib.CompileTimeAssemblies.Select(e => e.Path)
```

Comment:
I'm confused...how can this start with bin?

Reply:
I think this comes from [here](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/LockFileUtils.cs#L486). The test projects are more like facades and dont really have files, which may be causing this. But thats a general test issue.